dspace-api/src/main/java/org/dspace/authorize/dao/impl/ResourcePolicyDAOImpl.java

```diff
@@ -192,6 +202,15 @@ public void deleteByDsoEPersonPolicies(Context context, DSpaceObject dso, EPerso
 
     }
 
+    @Override
+    public void deleteAllEPersonPolicies(Context context, EPerson ePerson) throws SQLException {
+        String queryString = "delete from ResourcePolicy where eperson= :eperson";
```

Comment:
It probably can. But in the same class there are a lot of comparable methods that look exactly the same. I did not change all of them to HQL as this would add code changes that have nothing to do with the deletion of EPersons. I would like to keep this PR as narrow as possible to reduce the risk of introducing new bugs. Shall we open up a separate ticket regarding the conversion of SQL to HQL?